### ISSUE TITLE
Displaying inference using Pytorch on webcam frame

### ISSUE THREAD

**Issue by glitchyordis** [ultralytics/yolov5#9115]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

System: Windows 10
Python: 3.9

Hi, how do I show the results (ie detection bb + label on image) from "results = model(frame, size=640)" on an OpenCV window, where frame is a webcam input?

Code to duplicate:
```
import torch
import time
import cv2
from PIL import Image
import numpy as np
import pathlib
gen_path = pathlib.Path.cwd()

DEVICE = "cuda" if torch.cuda.is_available() else "cpu" 
  
model = torch.hub.load('ultralytics/yolov5', 'yolov5s')
cam = cv2.VideoCapture(0)
  
while(True): 
    ret, frame = cam.read()
    frame = frame[:, :, [2,1,0]]
    frame = Image.fromarray(frame) 
    frame = cv2.cvtColor(np.array(frame), cv2.COLOR_RGB2BGR)

    result = model(frame,size=640)
    cv2.imshow('Output', result)

    if cv2.waitKey(1) & 0xFF == ord('q'):
        break
  
cam.release()
cv2.destroyAllWindows()
```

Code Error:
![image](https://user-images.githubusercontent.com/99534415/186309567-e1849357-fa08-41da-a7f3-f170caf0a99c.png)

```
error: OpenCV(4.6.0) :-1: error: (-5:Bad argument) in function 'imshow'
> Overload resolution failed:
>  - mat is not a numpy array, neither a scalar
>  - Expected Ptr<cv::cuda::GpuMat> for argument 'mat'
>  - Expected Ptr<cv::UMat> for argument 'mat'
```

**Comment by MartinPedersenpp** [comment]
Take a look here:
https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading#issue-636702522

The result from the torch hub model is not an image, but a object.
You can either get the x,y coordinates and draw the boxes yourself with opencv or you can use results.render() to render the boxes on the frame that was fed to the model.

**Comment by glitchyordis** [comment]
@MartinPedersenpp 

Thanks for the response. For those wondering, you can use "cv2.imshow('YOLO',  np.squeeze(results.render()))".
